style-guide/styles/wds-components/_banner-notification.scss

```diff
@@ -42,22 +42,6 @@
 		font-size: $wds-typescale-size-minus-1;
 		line-height: 1.29;
 		padding: 13px 12px;
-
-		.wds-alert & a {
```

Comment:
won't the lack of these styles break styling in Feeds and all the applications except UCP?